meta/main.yml

```diff
@@ -1,4 +1,4 @@
----
+  ---
```

Comment:
not sure how pedantic the YaML parser is but I think you need three dashes alone on the line.